dist/index.d.ts

```diff
@@ -69,6 +68,6 @@ declare module "@mapbox/shp-write" {
 
   export function zip<T extends OutputType>(
     geojson: GeoJSON.FeatureCollection,
-    options: DownloadOptions & ZipOptions = {},
-    stream = false): Promise<OutputByType[T]>;
+    options: DownloadOptions & ZipOptions,
```

Comment:
This looks good, but this argument and `stream` should theoretically be optional in both methods, so can you change that too while you're at it? Thanks!